content/docs/setup/kubernetes/install/helm/index.md

```diff
@@ -193,19 +163,59 @@ to manage the lifecycle of Istio.
     58
     {{< /text >}}
 
-1. Install the `istio` chart:
+1. Install the `istio` chart. For a production environment use the recommended Helm values or customize as needed:
 
     {{< text bash >}}
-
     $ helm install istio.io/istio --name istio --namespace istio-system
+    {{< /text >}}
+
+    {{< tip >}}
+    To try the `bookinfo` demo, use Helm values customized for the `bookinfo` application:
+    {{< /tip >}}
 
+    {{< text bash >}}
+    $ helm install istio --name istio --values install/kubernetes/helm/istio/values-istio-demo.yaml --namespace istio-system
     {{< /text >}}
 
-1. Uninstall steps:
+## Uninstall
+
+1. If you installed using Option 1 with `helm template`, uninstall with this command:
 
     {{< text bash >}}
+    $ kubectl delete namespace istio-system
+    {{< /text >}}
 
+1. If you installed using Option 2 and `Tiller`, uninstall with these commands:
+
+    {{< text bash >}}
     $ helm delete --purge istio
     $ helm delete --purge istio-init
+    {{< /text >}}
+
+## Deleting CRDs and Istio Configuration
+
+{{< tip >}}
+Istio, by design, expects Istio's Custom Resources contained within CRDs to leak into the
+Kubernetes environment. CRDs contain the runtime configuration set by the operator.
+Because of this, we consider it better for operators to explicitly delete the runtime
+configuration data rather than unexpectedly lose it.
+{{< /tip >}}
+
+{{< warning >}}
+Deleting CRDs permanently deletes any configuration changes that you have made to Istio.
+{{< /warning >}}
+
+{{< tip >}}
```

Comment:
Do not use tips for information needed to understand the instructions. Tips are intended for optional information which would enhance the readers' understanding of the procedure, not the project.

Reply:
That was the original content, how come it was committed and approved?

Reply:
Removed the <tip>. For reference, it was introduced in different PR https://github.com/istio/istio.io/pull/3276/files, submitted by owner and approved by owner. We need clear guidelines and same for everybody.